DTS-STN/passport-status
# Updated ESRF validation

## [ADO-1282](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1282)

### Description
This PR adds additional validation to the ESRF form field based on newest requirements. In these requirements it is stated that the ESRF must begin with a letter (Examples of possible formats: A1234567 or AA123456 or A123456 or AA12345). The requirements state that we only need to check for the first character, so we don't have to check for all possible formats, just that it begins with a letter. I've altered the mocks to reflect this for the time being until new mocks are generated just so Cypress tests can pass. 

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/status`
4. Enter `12345678` and confirm that an error pops up stating the proper ESRF format
